cmd/migrate/migrate.go

```diff
@@ -39,7 +39,7 @@ func NewMigrateCommand() *cobra.Command {
 	flags.String(datastoreEngineFlag, "", "(required) the datastore engine that will be used for persistence")
 	flags.String(datastoreURIFlag, "", "(required) the connection uri of the database to run the migrations against (e.g. 'postgres://postgres:password@localhost:5432/postgres')")
 	flags.Uint(versionFlag, 0, "the version to migrate to (if omitted the latest schema will be used)")
-	flags.Duration(timeoutFlag, 1*time.Minute, "a timeout after which the migration process will terminate")
+	flags.Duration(timeoutFlag, 1*time.Minute, "a timeout for the time it takes the migrate process to connect to the DB")
```

Comment:
Please avoid the acronym here and just say "database"

Reply:
@jon-whit sure thing, updated, and also rebased the branch